README.md

```diff
@@ -195,6 +195,57 @@ buildctl build \
 
 `--local` exposes local source files from client to the builder. `context` and `dockerfile` are the names Dockerfile frontend looks for build context and Dockerfile location.
 
+##### Hint: `dockerfile.v0` frontend with `buildctl build --opt` option
+For `dockerfile.v0` frontend there are some parameters that can be used with `--opt`.
+The following parameters are the most frequently used.
+* target: set the target build stage to build.
+  * Example: `target=Stage1`
+* filename: name of the Dockerfile (Default is 'Dockerfile'),set to use a differently-named Dockerfile.
+  * Example: `filename=dockerfile.prod`,`filename=other.dockerfile`
+* dockerfilekey: dir path of the custom named context for Dockerfile. Useful when there is more than one build source.
+  * Full command example: 
+    * `buildctl --frontend=dockerfile.v0 --local dockerfile=. --local dockerfile2=/tmp dockerfilekey=dockerfile2` buildctl will try to use the `/tmp/Dockerfile` as Dockerfile.
+* context: define additional build context with specified contents. In Dockerfile the context can be accessed when `FROM` name or `--from=name` is used. When Dockerfile defines a stage with the same name it is overwritten.
+  * When using multiple contexts, use ":" to name the different contexts
+  * Container Image: `context:alpine=docker-image://alpine`
+  * Git: `context=git@github.com:moby/buildkit`,``
+  * Local source directory: `context=.`,`context:mydir=/path/to/dir`
+  * URL: `context=https://github.com/moby/moby.git` 
+  * Full command example:
+    ```shell
+    buildctl build \
+        --frontend=dockerfile.v0 \
+        --local dockerfile=. \
+        --local context=. \
+        --local mydir_local_is_tmp=/tmp \
+        --opt context:image-base=docker-image://alpine \
+        --opt context:mydir=local:mydir_local_is_tmp \
+        --opt context:git-base=git@github.com:moby/buildkit \
+        --opt context:http-base=https://github.com/moby/moby.git
+    ```
+    ```dockerfile
+    FROM image-base
+    COPY --from=mydir myfile /
+    FROM git-base
+    FROM http-base
+    ```
+* source: the build based image.Example `source="docker/dockerfile:master` 
```

Comment:
I don't quite understand this one. Does it replace the `# syntax` directive?

Reply:
yes. I updated it's description --> the location of the Dockerfile syntax that is used to build the Dockerfile.